### PR TITLE
refactor[w1.4]: modified feedback class

### PR DESCRIPTION
**Pull request by jelenazeljko55**

### CODE REVIEW

**Comment by jelenazeljko55** [comment]
[(3) W1.4 : As a website user I want to observe feedbacks so that I could find out if the hospital is suitable for me.](https://trello.com/c/stvahRMG/4-3-w14-as-a-website-user-i-want-to-observe-feedbacks-so-that-i-could-find-out-if-the-hospital-is-suitable-for-me)

File: HealthcareSystem/Backend/Model/Users/Feedback.cs [hunk 1]
```diff
@@ -18,6 +18,8 @@ public class Feedback
     {
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public DateTime SendingDate { get; set; }
```
Comment (vlaksi) [inline HealthcareSystem/Backend/Model/Users/Feedback.cs:22]: Why is SendingDate DatabaseGeneratedOption.Identity ?

Comment (jelenazeljko55) [reply HealthcareSystem/Backend/Model/Users/Feedback.cs:22]: https://docs.microsoft.com/en-us/ef/core/modeling/generated-properties?tabs=data-annotations&fbclid=IwAR0__ecVDglOZEhuipolDDMPXQWNOLz-wK9RD5Grd8i0Au9x3CiqOnVAvXk

**Review by vlaksi — DISMISSED** [review]

**Review by milana-todorovic — DISMISSED** [review]
Seems alright, just solve conflicts.

**Review by vlaksi — APPROVED** [review]

**Review by milana-todorovic — APPROVED** [review]

**Pull request merged**